core/src/main/scala/org/apache/spark/memory/MemoryManager.scala

```diff
@@ -155,4 +164,217 @@ private[spark] abstract class MemoryManager extends Logging {
     _storageMemoryUsed
   }
 
+  // -- The code formerly known as ShuffleMemoryManager --------------------------------------------
```

Comment:
Sure. I'll still leave a pointer to the ShuffleMemoryManager name just in case anyone grepping through the new codebase is curious about what happened to it.